org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/ASTParser.java

```diff
@@ -1136,11 +1135,8 @@ private ASTNode internalCreateAST(IProgressMonitor monitor) {
 								}
 							} catch(JavaModelException e) {
 								// an error occured accessing the java element
-								StringWriter stringWriter = new StringWriter();
-								try (PrintWriter writer = new PrintWriter(stringWriter)) {
-									e.printStackTrace(writer);
-								}
-								throw new IllegalStateException(String.valueOf(stringWriter.getBuffer()));
+								CharSequence stackTrace = org.eclipse.jdt.internal.compiler.util.Util.getStackTrace(e);
```

Comment:
was this automated cleanup? looks odd to not use an import here.

Reply:
No all in the first commit was done by hand. But in this class there is already
 `import org.eclipse.jdt.internal.core.util.Util;`
which is why `org.eclipse.jdt.internal.compiler.util.Util` cannot be imported.